src/apps/lwaftr/lwutil.lua

```diff
@@ -18,6 +19,19 @@ local ehs = constants.ethernet_header_size
 local o_ipv4_flags = constants.o_ipv4_flags
 local ntohs = lib.ntohs
 
+-- Produces configuration for each instance.
```

Comment:
Can we put this in `program.lwaftr.setup` instead?  This lwutils file is a little weird and I'd like to avoid adding to it.